interfaces/uss_qualifier/uss_qualifier.yaml

```diff
@@ -1,198 +1,263 @@
-          example: 'Started|Finished'
-        task_result:
-          type: object
-          description: >-
-            Tests result if task has completed.
-        filename:
-          type: string
-          description: >-
-            filename if test result is saved in an output report file.
-    FileUploadResponse:
-      type: object
-      required:
-      - status_message
-      properties:
-        status_message:
-          type: string
-          description: >-
-            Returns current status of uploaded files.
+    schemas:
+        TestRunSpecification:
+            type: object
+            properties:
+                flight_records:
+                    description: >-
+                        Comma separated list of flight record file names.
+                    type: string
+                auth_spec:
+                    description: >-
+                        User authentication specification.
+                    type: string
+                user_config:
+                    description: >-
+                        User authentication configuration.
+                    type: string
+        CreateTestRunRequest:
+            type: object
+            required:
+                - flight_records
+                - auth_spec
+                - user_config
+            properties:
+                flight_records:
+                    description: >-
+                        Comma separated list of flight record file names. Files should be uploaded by upload method before running the tests.
+                    type: string
+                auth_spec:
+                    description: >-
+                        User authentication specification.
+                    type: string
+                user_config:
+                    description: >-
+                        User authentication configuration.
+                    type: string
+        BackgroundTaskRunResponse:
+            type: object
+            properties:
+                task_id:
+                    $ref: "#/components/schemas/TaskID"
+                status_message:
+                    type: string
+                    example: "A task has been started in the background."
+                    description: >-
+                        Status of currently running task.
+        CreateTestRunResponse:
+            type: object
+            properties:
+                task_id:
+                    $ref: "#/components/schemas/TaskID"
+                status_message:
+                    type: string
+                    example: "A task has been started in the background."
+                    description: >-
+                        Status of currently running task.
+                user_id:
+                    type: string
+                    description: >-
+                        User ID of user running the tests.
+                specification:
+                    $ref: "#/components/schemas/TestRunSpecification"
+        TestsRunResponse:
```

Comment:
I would expect that the representation of a `test_run` resource should be essentially the same whether it is being reported as a consequence of creation, or being queried specifically.  I think `CreateTestRunResponse` is basically what we want here (though we should probably rename it `TestRunResponse` to logically use it in both places), after adding an optional `outputs` field (Or perhaps `report`?  Are there are outputs apart from the report?) which would be populated only after the test run completed.